website/addons/owncloud/static/owncloudNodeConfig.js

```diff
@@ -0,0 +1,315 @@
+'use strict';
+
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+var $osf = require('js/osfHelpers');
+var $modal = $('#ownCloudCredentialsModal');
+var oop = require('js/oop');
+var OauthAddonFolderPicker = require('js/folderPickerNodeConfig');
+var language = require('js/osfLanguage').Addons.owncloud;
+
+
+var ViewModel = oop.extend(OauthAddonFolderPicker,{
+    constructor: function(addonName, url, selector, folderPicker, opts){
+        var self = this;
+        self.super.constructor.call(self, addonName, url, selector, folderPicker);
+        const otherString = 'Other (Please Specify)';
+
+        self.addonName = 'ownCloud';
```

Comment:
I think everything between here and `self.hosts` is handled by the superclass. Please remove; there's no reason to duplicate these assignments.